railgun/cmd/rootcmd/env.go

```diff
@@ -22,7 +23,22 @@ func bindEnvVars(cmd *cobra.Command, _ []string) {
 		}
 
 		if envValue, envSet := os.LookupEnv(envKey); envSet {
-			cmd.Flags().Set(f.Name, envValue)
+			if err := f.Value.Set(envValue); err != nil {
+				errs = append(errs, fmt.Errorf("failed to set value for %s: %w", envKey, err))
+			}
 		}
 	})
+
+	var wrappedErr error
+	if len(errs) > 0 {
+		for i, err := range errs {
+			if i == 0 {
+				wrappedErr = err
+			} else {
+				wrappedErr = fmt.Errorf("%v: %w", wrappedErr, err)
+			}
+		}
```

Comment:
this error assembling functionality occupies more than 50% of this function and is untested.

Please consider doing one of the following:
- just `return err` on the first error in this function
- implement the "wrap all errors in one **über**error" functionality as a util function (probably outside of this package) and test it

Reply:
Ok, changed to just return a single error in 34a05c004c2dcee8be690fed215dbd0afcac4654 :+1: